softdevice_controller/CHANGELOG.rst

```diff
@@ -24,13 +24,19 @@ Changes
 =======
 
 * When LLPM mode is enabled, the connection event length is now no longer implicitly set to 1 ms.
+* When the connection interval is a LLPM connection interval, that is, below 7.5 ms, link layer procedures with an instant will use an instant larger than 6.
```

Comment:
This seems unrelated?

```suggestion
* When the connection interval is an LLPM connection interval, that is, below 7.5 ms, link layer procedures with an instant will use an instant larger than 6.
```
Not sure what an instant is, but should it be "6 ms"?

Reply:
I will change it to 6 connection events